### ISSUE TITLE
Azure Devops Pipeline issue: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

### ISSUE THREAD

**Issue by ahamedfazil** [Azure/static-web-apps#555]
Hi,
I am using Azure Static webapp with Nextjs and setup my project in Azure Devops. The pipeline was working fine but now I started getting below error.

Here is the code of my azure-pipeline.yml

`trigger:
  - develop
  - main

pool:
  vmImage: 'ubuntu-latest'

steps:
  - checkout: self
    submodules: true

  - task: AzureStaticWebApp@0
    inputs:
      app_location: 'my-app'
      api_location: 'api'
      output_location: 'out'
      azure_static_web_apps_api_token: $(deployment_token)`

**Screenshots**
This is the error I get during build. But it is working fine locally

![image](https://user-images.githubusercontent.com/21005625/130353339-30988083-7320-446a-9a05-256cece816a5.png)

I am not sure what went wrong.

**Comment by anthonychu** [comment]
Can you please share your package.json contents? Also if there was a commit that was building correctly before, can you re-run the build on that to confirm the behavior has changed on the same app?

**Comment by ahamedfazil** [comment]
this is my package.json

{
  "name": "my-app",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build && next export",
    "start": "next start",
    "lint": "next lint",
    "export": "next export",
    "api": "func start --prefix ../api",
    "swa": "swa start http://localhost:3000 --api http://localhost:7071"
  },
  "dependencies": {
    "bootstrap": "5.1.0",
    "framer-motion": "4.1.17",
    "next": "11.1.0",
    "react": "17.0.2",
    "react-bootstrap": "1.6.1",
    "react-dom": "17.0.2"
  },
  "devDependencies": {
    "@types/node": "16.6.1",
    "@types/react": "17.0.11",
    "eslint": "7.32.0",
    "eslint-config-next": "11.1.0",
    "typescript": "4.3.5",
    "sass": "1.35.1"
  }
}


In the below screenshot is taken from working build in azure devops.

![image](https://user-images.githubusercontent.com/21005625/130366423-f79ae7ee-b935-4eb8-a059-df1aa4fa3d8c.png)


During that build also I notice fsevents error but that was optional dependency and it throw it as a warning not error
![image](https://user-images.githubusercontent.com/21005625/130366527-16608858-9dc4-4f50-b2cd-62e1a92ee072.png)

**Comment by anthonychu** [comment]
Yep that's typically a warning. I just tried that package.json and it installed with a warning. I wonder if this was caused by something the package-lock.json.

If you rerun the build using the commit that was working before, does it still work?

**Comment by ahamedfazil** [comment]
If I rerun the commit that was working, it still works with warning.

I just tested by removed the package-lock in the commit in which it throws error. And now that also working. I guess my package-lock is somehow messed up.
